nixos/modules/services/web-apps/isso.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption types literalExample;
+
+  cfg = config.services.isso;
+
+  settingsFormat = pkgs.formats.ini { };
+  configFile = settingsFormat.generate "isso.conf" cfg.settings;
+in {
+
+  options = {
+    services.isso = {
+      enable = mkEnableOption ''
+        Whether to enable isso, a commenting server similar to Disqus.
+
+        Note: The application's author suppose to run isso behind a reverse proxy.
+        The embedded solution offered by NixOS is also only suitable for small installations
+        below 20 requests per second.
+      '';
+
+      user = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service user.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service group.";
+      };
+
+      settings = mkOption {
+        description = ''
+          Configuration for <package>isso</package>.
+
+          See <link xlink:href="https://posativ.org/isso/docs/configuration/server/">Isso Server Configuration</link>
+          for supported values.
+        '';
+
+        type = types.submodule {
+          freeformType = settingsFormat.type;
+        };
+
+        default = { };
+
+        example = literalExample ''
+          {
+            general = {
+              dbpath = "/tmp/isso-comments.db";
```

Comment:
This seems like a bad example... Shouldn't you add `StateDirectory` to your `serviceConfig` and set the value of `dbpath` to use that directory?

Reply:
Added `StateDirectory` and point example path to that directory.